specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-04-01-preview/workspaceRP.json

```diff
@@ -3321,6 +4100,10 @@
         "provisioningState": {
           "description": "Read-only provision state status property.",
           "$ref": "#/definitions/DefaultResourceProvisioningState"
+        },
+        "shouldCreateAiServicesEndpoint": {
+          "description": "Whether the proxy (non-byo) endpoint is a regular endpoint or a OneKeyV2 AI services account endpoint.",
```

Comment:
consider using enum instead of boolean

Reply:
this has already been released and used by UX, will consider that in the future API design, but will keep it as boolean this time.